es/index.html

```diff
@@ -32,6 +32,7 @@
 	<link rel="mask-icon" href="/safari-pinned-tab.svg" color="#5bbad5">
 	<meta name="msapplication-TileColor" content="#da532c">
 	<meta name="theme-color" content="#ffffff">
+	<meta itemprop="datePublished" content="2019-07">
```

Comment:
Don't we also need copyright?

Reply:
Yes I added. 

![copyrightYear](https://user-images.githubusercontent.com/29107510/79503912-f579a700-7fff-11ea-8c9b-895e786e25d2.png)

Reply:
Ah whoops! I didn't have other commits when I reviewed it for some reason. My bad!